docs/component-guides/pipelines.md

```diff
@@ -66,32 +67,15 @@ client = kfp.Client(host=f"https://{kubeflow_gateway_endpoint}/pipeline", cookie
 client.list_experiments(namespace=namespace)
 ```
 
-## S3 Access from Kubeflow Pipelines
-
-It is recommended to use AWS credentials to manage S3 access for Kubeflow Pipelines. [IAM Role for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) requires applications to use the latest AWS SDK to support the `assume-web-identity-role`. This requirement is in development, and progress can be tracked in the [open GitHub issue](https://github.com/kubeflow/pipelines/issues/3405).
-
-A Kubernetes Secret is required by Kubeflow Pipelines and applications to access S3. Be sure that the Kubernetes Secret has S3 read and write access.
+## AWS Access from Kubeflow Pipelines
 
-```
-apiVersion: v1
-kind: Secret
-metadata:
-  name: aws-secret
-  namespace: kubeflow
-type: Opaque
-data:
-  AWS_ACCESS_KEY_ID: <YOUR_BASE64_ACCESS_KEY>
-  AWS_SECRET_ACCESS_KEY: <YOUR_BASE64_SECRET_ACCESS>
-```
+User profiles can be granted permissions to access AWS resources. Pipelines under the profile namespace will have access to AWS resources as specified in the user profile's `awsIamRole`.
 
-- YOUR_BASE64_ACCESS_KEY: Base64 string of `AWS_ACCESS_KEY_ID`
-- YOUR_BASE64_SECRET_ACCESS: Base64 string of `AWS_SECRET_ACCESS_KEY`
+The instructions for managing AWS permissions for user profiles can be found [here](/kubeflow-manifests/docs/component-guides/profiles). The below [Example Pipeline](#example-pipeline) can be used to verify the configuration.
```

Comment:
Might be a little unclear/hard for customer to understand here. These instructions are necessary to follow to setup KFP now since we are recommending this IRSA approach

Reply:
yeah I agree with @ryansteakley  here lets have an example with e2e setup. you can link it for more details but lets add here. 

Something like setup user profile with such permissions which will be inherited and used by pipeline or something along that line.

Reply:
done